components/actionbar/index.css

```diff
@@ -10,50 +10,174 @@ governing permissions and limitations under the License.
+  &:lang(ko) {
+    --spectrum-actionbar-item-counter-line-height-cjk : var(--mod-CJK-line-height-100, var(--spectrum-CJK-line-height-100));
+  }
+
+  /* colors - applied to popover */
+  --spectrum-actionbar-popover-background-color: var(--mod-gray-50, var(--spectrum-gray-50));
+  --spectrum-actionbar-popover-border-color: var(--mod-gray-400, var(--spectrum-gray-400));
+
+  /* emphasized variation colors */
+  --spectrum-actionbar-emphasized-background-color : var(--mod-informative-background-color-default, var(--spectrum-informative-background-color-default));
+  --spectrum-actionbar-emphasized-item-counter-color: var(--mod-white, var(--spectrum-white));
+
+  /* spacing of action bar bottom outer edge */
+  --spectrum-actionbar-spacing-outter-edge: var(--mod-spacing-300, var(--spectrum-spacing-300));
+
+  /* spacing of close button */
+  --spectrum-actionbar-spacing-close-button-top: var(--mod-spacing-100, var(--spectrum-spacing-100));
+  --spectrum-actionbar-spacing-close-button-start : var(--mod-spacing-100, var(--spectrum-spacing-100));
+  --spectrum-actionbar-spacing-close-button-end: var(--mod-spacing-75, var(--spectrum-spacing-75));
+
+  /* spacing of item counter field label */
+  --spectrum-actionbar-spacing-item-counter-top : var(--mod-action-bar-top-to-item-counter, var(--spectrum-action-bar-top-to-item-counter));
+  --spectrum-actionbar-spacing-item-counter-end: var(--mod-spacing-400, var(--spectrum-spacing-400));
+
+  /* spacing of action group */
+  --spectrum-actionbar-spacing-action-group-top: var(--mod-spacing-100, var(--spectrum-spacing-100));
+  --spectrum-actionbar-spacing-action-group-end: var(--mod-spacing-100, var(--spectrum-spacing-100));
+
+  /* drop shadow */
+  --spectrum-actionbar-shadow-horizontal : var(--mod-drop-shadow-x, var(--spectrum-drop-shadow-x));
+  --spectrum-actionbar-shadow-vertical : var(--mod-drop-shadow-y, var(--spectrum-drop-shadow-y));
+  --spectrum-actionbar-shadow-blur : var(--mod-drop-shadow-blur, var(--spectrum-drop-shadow-blur));
+  --spectrum-actionbar-shadow-color : var(--mod-drop-shadow-color, var(--spectrum-drop-shadow-color));
 }
 
-.spectrum-ActionBar {
-  display: flex;
-  justify-content: center;
+/* windows high contrast mode */
+@media (forced-colors: active) {
+  .spectrum-ActionBar {
+    --highcontrast-actionbar-popover-border-color: CanvasText;
+  }
 
+  .spectrum-ActionBar--emphasized {
+    .spectrum-ActionBar-popover {
+      --highcontrast-actionbar-popover-border-color: CanvasText;
+    }
+  }
+}
+
+/* ActionBar is outer wrapper with nested popover component within */
+.spectrum-ActionBar {
+  margin-block-end: var(--mod-actionbar-spacing-outter-edge, var(--spectrum-actionbar-spacing-outter-edge));
   inset-block-end: 0;
   z-index: 1;
 
   /* Account for fixed width */
   box-sizing: border-box;
 
-  /* Restrict the width of the child popover */
-  padding: 0 var(--spectrum-actionbar-margin-x);
+  /* Let clicks in blank space fall through */
+  pointer-events: none;
```

Comment:
oh interesting!

Reply:
I did not write this bit, just moved it.